ui/job-view/details/PinBoard.jsx

```diff
@@ -568,71 +578,64 @@ class PinBoard extends React.Component {
             className="btn-group-vertical"
             title={this.hasPinnedJobs() ? '' : 'No pinned jobs'}
           >
-            <div className="btn-group save-btn-group dropdown">
+            <ButtonGroup className="save-btn-group">
```

Comment:
Cripes...  My apologies for not converting things like these when I re-wrote this from Angular...